templates/candlepin.conf.erb

```diff
@@ -26,9 +26,9 @@ candlepin.ca_key_password=<%= scope['candlepin::ca_key_password_unsensitive'] %>
 <%- end -%>
 
 candlepin.async.jobs.ExpiredPoolsCleanupJob.schedule=<%= scope['candlepin::expired_pools_schedule'] %>
+<% if scope['candlepin::loggers'].any? -%>
```

Comment:
I am surprised this `any?` is needed.

Reply:
Mostly for a visual separation with an empty line between the groups.